### PR TITLE
Test legacy jit CI workflow

### PR DESCRIPTION
**Pull request by rartemev**

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Comment by rartemev** [comment]
@dotnet-bot test x86_lb_checked_windows_nt_bld please

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT x86lb Debug Build and Test

File: netci.groovy [hunk 1]
```diff
@@ -1459,7 +1484,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                             Utilities.addArchival(newJob, "stdout.txt")
                             Utilities.addArchival(newJob, "Logs/**")
                         }
-                        else if (architecture == 'x64' || architecture == 'x86') {
+                        else if (architecture == 'x64' || architecture == 'x86' || architecture == 'x86lb') {
                             buildCommands += "tests\\runtest.cmd ${runtestArguments}"
```
Comment (jashook) [inline netci.groovy]: runtest.cmd is most likely not going to understand x86lb as a architecture. There used to be some logic either to use a created TestEnv or use a checked in env.

At a minumum the test env will need to define:

COMPlus_AltJit=*
COMPlus_AltJitName=legacyjit

**Comment by rartemev** [comment]
@dotnet-bot test ci please

**Comment by rartemev** [comment]
@dotnet-bot test ci please

**Comment by jashook** [comment]
Just note that you will need to work with issues.targets to properly exclude for only legacy_backend. That does not have to happen in this pr.

**Pull request merged**